### ISSUE TITLE
欧式距离

### ISSUE THREAD

**Issue by ccyycc-hash** [Li-ZK/DCFSL-2021#2]
看您的代码里距离是logits = -((a - b) ** 2).sum(dim=2)  ，看这里不是欧式距离。。求教 为啥这样写。
